### PR TITLE
Update dependency go modules for k8s v1.26.0-rc.0

### PR DESCRIPTION
**Pull request by sunnylovestiramisu**
Ran kubernetes-csi/csi-release-tools go-get-kubernetes.sh -p 1.26.0-rc.0

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -59,23 +59,41 @@ require (
...
-	k8s.io/kube-openapi v0.0.0-20220803164354-a70c9af30aea // indirect
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed // indirect
+	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280 // indirect
+	k8s.io/utils v0.0.0-20221107191617-1a15be271d1d // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace k8s.io/api => k8s.io/api v0.26.0-rc.0
```
Comment (xing-yang) [inline go.mod:83]: Do we still need all these "replace" lines?

Comment (sunnylovestiramisu) [reply go.mod:83]: @msau42 suggested us keeping the replace lines for now.

**Comment by sunnylovestiramisu** [comment]
The failure informaion:
```
pkg/capacity/apibridge.go:101:9: cannot use csiStorageCapacityIndexInformerBridge{…} (value of type csiStorageCapacityIndexInformerBridge) as type "k8s.io/client-go/tools/cache".SharedIndexInformer in return statement:
	csiStorageCapacityIndexInformerBridge does not implement "k8s.io/client-go/tools/cache".SharedIndexInformer (wrong type for AddEventHandler method)
		have AddEventHandler("k8s.io/client-go/tools/cache".ResourceEventHandler)
		want AddEventHandler("k8s.io/client-go/tools/cache".ResourceEventHandler) ("k8s.io/client-go/tools/cache".ResourceEventHandlerRegistration, error)
pkg/capacity/apibridge.go:116:35: cannot use csiStorageCapacityIndexInformerBridge{} (value of type csiStorageCapacityIndexInformerBridge) as type "k8s.io/client-go/tools/cache".SharedIndexInformer in variable declaration:
	csiStorageCapacityIndexInformerBridge does not implement "k8s.io/client-go/tools/cache".SharedIndexInformer (wrong type for AddEventHandler method)
		have AddEventHandler("k8s.io/client-go/tools/cache".ResourceEventHandler)
		want AddEventHandler("k8s.io/client-go/tools/cache".ResourceEventHandler) ("k8s.io/client-go/tools/cache".ResourceEventHandlerRegistration, error)
```
https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/capacity/apibridge.go#L100
https://github.com/kubernetes/client-go/blame/master/tools/cache/shared_informer.go#L513

There is no recent change

**Comment by msau42** [comment]
It looks like the interface changed here: https://github.com/kubernetes/client-go/commit/ecdc8bf7296ca836be500025e53a1b27506e3a6c

**Pull request merged**